learn-data-structures-by-building-a-shopping-cart/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>freeCodeCamp Shopping cart</title>
```

Comment:
This could be in title case:

```suggestion
    <title>freeCodeCamp Shopping Cart</title>
```